scripts/sample-reads-randomly.py

```diff
@@ -86,7 +86,8 @@ def get_parser():
 
     parser.add_argument('filenames', nargs='+')
     parser.add_argument('-N', '--num_reads', type=int, dest='num_reads',
-                        default=DEFAULT_NUM_READS)
+                        default=DEFAULT_NUM_READS, help='samples the number of sequences' 
+                        'or pairs specified with -N')
```

Comment:
If you concatenate the two halves of the string, there's no space between "of sequences" and "or pairs".

Sorry to be so nitpicky. I can only do so after having made that mistake many...many...many times myself: :-)